packages/hooks/src/configure.test.ts

```diff
@@ -0,0 +1,19 @@
+import { Configuration, configure } from "./configure";
+
+describe("configure", () => {
+  const originalNotifier = Configuration.errorNotifier;
+  afterEach(() => {
+    configure({
+      errorNotifier: originalNotifier,
+    });
+  });
+
+  it("should set the errorNotifier to the passed in configuration option", () => {
+    const myNotifier = jest.fn();
+    configure({
+      errorNotifier: myNotifier,
+    });
+
+    expect(Configuration.errorNotifier).toBe(myNotifier);
+  });
```

Comment:
since `configure` has not been used anywhere, I wonder if we need this test.

Reply:
See this comment. https://github.com/GetJobber/atlantis/pull/561#discussion_r657291605